polling_stations/apps/addressbase/models.py

```diff
@@ -70,6 +70,12 @@ class Meta:
         db_column="uprn",
     )
     polling_station_id = models.CharField(blank=True, max_length=255)
+    advance_voting_station = models.ForeignKey(
```

Comment:
Again, not tested - does this FK constraint have any impact on the time it takes to do an addressbase import, or not really?

Reply:
I've not tested on a full import (I'm just using the testing data), so we'll have to investigate. My assumption that it would make very little difference. I don't know but I reckon the clever folks at Postgres have optimized for inserting nothing into a nullable column. (not a sarcastic comment...I really think they'll have done a better job at this than we can do)